openapi-v1.yaml

```diff
@@ -2632,6 +2632,113 @@ definitions:
       - array:read-write
       - array:admin
 
+  GroupAttributes:
+    description: Attributes describe the group itself, not any of the subgroups or assets
+    type: object
+    properties:
+      id:
+        description: the globally unique id of the group
+        type: string
+        x-omitempty: true
+      namespace:
+        description: The namespace of the group
+        type: string
+      name:
+        description: The name of the group. It is unique within the namespace. No 2 groups can have the same name
+        type: string
+      path:
+        description: A slash separated string of group names from the root to this group. Analogous to an OS filepath
+        type: string
+      description:
+        description: A human readable description of the content of the group
+        type: string
+        x-omitempty: true
+    example:
+      id: 0000-0001-0002-0003
+      namespace: my-organization
+      name: intro-to-genomics
+      path: /genomics-course/intro-to-genomics
+      description: contains arrays and notebooks for an 101 course on genomics with TileDB
+
+  GroupMember:
```

Comment:
Since this is an asset can we make it `GroupAsset`?

Reply:
Or better `GroupAssetCreate`

Reply:
This object is removed because of the changes in the endpoints